lib/tasks/anonymize.rake

```diff
@@ -86,7 +148,16 @@ namespace :anonymize do
 
       table 'patients' do
         primary_key 'id'
-        whitelist 'full_name', 'age', 'gender', 'status', 'address_id', 'date_of_birth'
+        scramble('patients', 'full_name')
+        scramble('patients', 'age')
```

Comment:
club age, dob, age_updated

Reply:
age / gender should be scramble?

Reply:
We should scramble age and gender since they can be used to identify patients along with BPs.